requirements/env_unidist_linux.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - pandas>=2.1,<2.2
   - numpy>=1.22.4
   - unidist-mpi>=0.2.1
+  - mpich
```

Comment:
I wonder why it is not a Linux-specific dependency of `unidist-mpi` instead...

Reply:
If we added any MPI library as a linux-specific dependency of `unidist-mpi`, we would jeopardize the option of using a system-provided MPI library.